com.google.EarthPro.xml

```diff
@@ -6,7 +6,8 @@
   <name>Google Earth Pro</name>
   <summary>3D planet viewer</summary>
   <description>
-    <p>NOTE: This is an unofficial package based on the official binaries releases at www.google.com/earth. Create maps with advanced tools: Google Earth Pro on desktop is free for users with advanced feature needs. Import and export GIS data, and go back in time with historical imagery</p>
+    <p>Create maps with advanced tools: Google Earth Pro on desktop is free for users with advanced feature needs. Import and export GIS data, and go back in time with historical imagery</p>
+    <p>NOTE: This is an unofficial package based on the official binaries releases at www.google.com/earth.</p>
```

Comment:
It should look like the disclaimer used for https://flathub.org/apps/com.jetbrains.PhpStorm

Reply:
Updated